lib/client/hooks/use-infinity-list.ts

```diff
@@ -91,16 +89,20 @@ function useInfinityList<Item>({
   )
   const [isLoading, setIsLoading] = useState(false)
 
-  console.log(response)
+  const prevSorting = usePrevious({ sortBy, sortOrder })
 
-  const data = response
+  let data = response
     ? [].concat(...response.map(({ results }) => results))
     : []
+  const prevData = usePrevious(data)
 
+  if (JSON.stringify(prevSorting) !== JSON.stringify({ sortBy, sortOrder }) && !response && prevData) {
+    data = prevData
```

Comment:
why all of this? 

This may incur in problems when you perform a load of another bulk of data before doing the new sorting.  I would remove this and let the list do the new fetching

Reply:
during useSWRInfinity lifecycle we can receive empty response and data will flash